keras_cv/layers/preprocessing_3d/input_format_test.py

```diff
@@ -15,98 +15,105 @@
 import tensorflow as tf
 from absl.testing import parameterized
 
+from keras_cv.backend.config import keras_3
 from keras_cv.layers import preprocessing_3d
 from keras_cv.layers.preprocessing_3d import base_augmentation_layer_3d
 from keras_cv.tests.test_case import TestCase
 
-POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
-BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+if not keras_3():
```

Comment:
is there a reason we can't just do this with the pytest annotation like elsewhere?

Reply:
The TEST_CONDITIONS list would also need to be moved inside the tests since those layers are also not compatible with Keras 3.  So left it as-is for now.  We can update it once we make them keras 3 compatible.